corehq/apps/enterprise/migrations/0003_enterprisepermissions_modify_account.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.24 on 2021-07-29 21:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0002_enterprisepermissions_account_unique'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='enterprisepermissions',
+            name='account',
+            field=models.OneToOneField(on_delete=django.db.models.deletion.CASCADE,
+                                       to='accounting.BillingAccount'),
+        ),
```

Comment:
I think this will have not change anything in the database. If that's true you can signal that to Django by putting it in `migrations.SeparateDatabaseAndState(state_operations=[...]` as in https://github.com/dimagi/commcare-hq/blob/db41d5aceb524a676149c113d6cc898dfdca58b6/corehq/blobs/migrations/0003_big_content.py#L20

Reply:
Thanks, updated in b83c466652c